GTBitsOfGood/southface
# ah/116-tag-selection-add-standard

## ah/116-tag-selection-add-standard

Issue Number(s): #116 

What does this PR change and why?

- Adds Tag Selection Component to the Add Standard Form (tried to use height styling to keep the columns as close to Figma as possible)
- Removed the previous frontend for viewing added tags (since they should just go into the main component)
- Also adds function for quickly inserting many tags into database

### Checklist

- [x] tag selection from Ricky's component is added to add-standard page (look at src/components/tag)
- [x] tag selection works alongside add to tag using input field


### How to Test

- Verify that selected tags add to standards correctly
- Verify that creating a new tag will add to the component w/o a refresh [not really sure why it is sometimes checked and sometimes not checked but did not think it was that big of a deal]
